app/scripts/controller/util.js

```diff
@@ -685,6 +685,28 @@
       return str;
     };
 
+    /**
+     * If input is na, NA, NaN, n/a, null or undefined, return true
+     * Else, return false.
+     * @param str
+     * @param includeEmptyStr Whether empty string should be treated as NA
+     * @returns {boolean}
+     */
+    content.strIsNa = function(str, includeEmptyStr) {
+      var status = false;
+      includeEmptyStr = _.isBoolean(includeEmptyStr) ? includeEmptyStr : false;
+      if (_.isString(str)) {
+        if (['na', 'nan', 'n/a'].indexOf(str.toLowerCase()) > -1) {
+          status = true;
+        } else if (includeEmptyStr && !str) {
+          status = true;
+        }
+      } else if (typeof str === 'undefined' || str === null) {
```

Comment:
We need it. The first if only handles String, this one only handles undefined and null. We don't deal with Number, Object or other types